### PR TITLE
Test that vertex attribute is preserved across context switches

### PR DESCRIPTION
**Pull request by y-novikov**
See http://crbug.com/920033 for rationale for this test.

### CODE REVIEW

**Comment by y-novikov** [comment]
@kenrussell 
I've checked that this fails with --use-gl=angle and passes with https://chromium-review.googlesource.com/c/angle/angle/+/1545009

File: sdk/tests/conformance/attribs/00_test_list.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ gl-vertexattribpointer-offsets.html
 --min-version 1.0.2 gl-vertex-attrib-render.html
 gl-vertex-attrib-zero-issues.html
 --min-version 1.0.4 gl-vertex-attrib-unconsumed-out-of-bounds.html
+--min-version 1.0.4 gl-vertex-attrib-context-switch.html
```
Comment (jdashg) [inline sdk/tests/conformance/attribs/00_test_list.txt]: Please try to keep this alphabetized.

File: sdk/tests/conformance/attribs/gl-vertex-attrib-context-switch.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+** IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+** CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+** TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+** MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+*/
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (jdashg) [inline sdk/tests/conformance/attribs/gl-vertex-attrib-context-switch.html]: Bad indentation.

**Comment by jdashg** [comment]
Thanks!

**Pull request merged**